Essentials/src/com/earth2me/essentials/UserData.java

```diff
@@ -510,6 +512,34 @@ public void setMuted(boolean set) {
         config.save();
     }
 
+    public String _getMuteReason() {
+        return config.getString("muteReason");
+    }
+
+    public String getMuteReason() {
+        if (muteReason != null) {
+            return muteReason;
+        }
+        else {
+            return "";
+        }
+    }
+
+    public void setMuteReason(String reason) {
+        if (reason.equals("")) {
+            config.removeProperty ("muteReason");
+            muteReason = null;
+        } else {
+            muteReason = reason;
+            config.setProperty ("muteReason", reason);
+        }
+        config.save();
+    }
+
+    public boolean hasMuteReason(){
+        return !getMuteReason().equals("");
```

Comment:
Is there any particular reason to call `getMuteReason` instead of just checking the value of `muteReason` directly?